app/models/document_type/multi_tag_field.rb

```diff
@@ -16,8 +16,4 @@ def pre_update_issues(_edition, _params)
   def pre_preview_issues(_edition)
     Requirements::CheckerIssues.new
   end
-
-  def pre_publish_issues(_edition)
-    Requirements::CheckerIssues.new
-  end
```

Comment:
>  Since the edition must satisfy all pre-preview issues before
> publishing, this removes the redundant pre-publish check.

Where is this in the code? I didn't know that.

Reply:
I guess it kind of makes sense, right? I can see why we'd want them to 'cascade', but they are actually separate, primarily because they are presented separately.

There's no actual code that 'enforces' this, but the user action to "Publish" isn't available unless the document can be previewed, and it can't be previewed unless the pre-preview requirements are met, which you can see in the following code:

   - https://github.com/alphagov/content-publisher/blob/master/app/services/failsafe_draft_preview_service.rb#L8
   - https://github.com/alphagov/content-publisher/blob/master/app/views/documents/show/_actions.html.erb#L25

Since the user probably doesn't want to publish a document that's out-of-date in the content store, it seems safe to assume they won't. There's [nothing technically stopping them](https://github.com/alphagov/content-publisher/blob/master/app/interactors/publish/publish_interactor.rb#L28), though.